index.d.ts

```diff
@@ -50,19 +54,20 @@ export interface RunCmd<A extends Action> {
   };
 }
 
-export interface SequenceCmd<A extends Action> {
-  type: 'Sequence';
-  cmds: CmdType<A>[];
-}
+//deprecated types
```

Comment:
@willisplummer do these type changes make sense? Since Cmd.batch and Cmd.sequence will now return instances of Cmd.list, it seemed like we could just make those types aliases of List.

Reply:
👍 makes sense to me